### DISCUSSION TITLE
Super-Resolution Reflectivity and Velocity

### DISCUSSION THREAD

**Discussion by mzuranski** [Unidata/gempak#67] [General]
Per [SCN 21-96](https://www.weather.gov/media/notification/pdf2/scn21-96_sbn_super-res.pdfl), Nexrad Reflectivity and Velocity products are getting an upgrade.  Attached are sample products issued by ROC in advance of the transition: [Sample_Super-Res.zip](https://github.com/Unidata/gempak/files/7998470/Sample_Super-Res.zip)

As of this posting RAX has already transitioned.  It has stopped transmitting the discontinued products and is transmitting the new Super-Res products.  I was able to get Gempak to plot this data by making the following additions to the tables (Note: I'm still on 7.5.1 but I didn't see these products in the latest version either):

In **~NAWIPS/gempak/tables/rad/nidprd.tbl** add the following lines:
LOCAL    BREF        153     256 DBZ        0.50 BASE_REFL_SUPERRES
LOCAL    VEL         154     256 KNOTS      0.50 RAD_VEL_SUPERRES

In **~NAWIPS/gempak/tables/sat/imgtyp.tbl** add the following lines:
LOCAL .50 KM  BASE    REFLECT      0    256      7    153      1 upc_dhr.tbl
LOCAL .50 KM  RADIAL VELOCITY      0    256      7    154      1 upc_vel.tbl

While this is able to make images for both of the new products, I'm not positive these are the best values.  I'm also getting odd behavior with the color bar, frequency seems to be ignored (see N0B sample, I used IMCBAR   = 31/V/CR/.98;.5/.84;.02/-10).  If I switch back to another product (e.g. N2U) the scale looks appropriate.

![test_N0B_gempak](https://user-images.githubusercontent.com/22502463/152436015-9f16925e-a4d2-4656-b53a-2072aedba0ee.gif)

**Comment by akrherz** [comment]
Thanks @mzuranski , are you able to submit a pull request with your suggested changes?

**Reply by mzuranski** [reply]
I'll see if I can make that happen this week.  I'll also try to test this on 7.14 beforehand to see if that changes anything about those color bar labels.

**Comment by sgdecker** [comment]
For posterity, the link for [SCN 21-96](https://www.weather.gov/media/notification/pdf2/scn21-96_sbn_super-res.pdf) is corrected here.
